src/numbers.js

```diff
@@ -1,45 +1,45 @@
 const add = (a, b) => {
-  // your code here
+  return a+b// your code here
 };
 
 const subtract = (a, b) => {
-  // your code here
+  return a-b// your code here
 };
 
 const multiply = (a, b) => {
-  // your code here
+  return a*b// your code here
 };
 
 const divide = (a, b) => {
-  // your code here
+  return a/b// your code here
 };
 
 const power = (a, b) => {
-  // your code here
+  return Math.pow (a,b)// your code here
```

Comment:
Spacing after function names is good when declaring methods or functions:

```js
function pow () {
  ...
}
```

But not when invoking:

```js
pow();
```